web/public/mgmt/config.yml

```diff
@@ -3671,6 +3671,8 @@ collections:
               - { label: Needs Account Modal Button, name: needsAccountModalButtonText, widget: string }
               - { label: Needs Account Modal Sub Button text, name: needsAccountModalSubText, widget: string }
               - { label: Add Business Button, name: addBusinessButton, widget: string }
+              - { label: My Account Text, name: navBarMyAccountText, widget: string }
+              - { label: Business.nj.gov Link, name: navBarBusinessNJGovHomePageLink, widget: string }
```

Comment:
[dust] I don't think we need to specify the home page. This could help in places where the text is a bit verbose. Consider changing the `name` to `navBarBusinessNJGovLink`.